cmd/lingo/main.go

```diff
@@ -142,6 +142,12 @@ func run() error {
 	autoscaler.Endpoints = endpointManager
 	go autoscaler.Start()
 
+	readiness := deployments.NewReadiness(mgr.GetClient(), namespace, deploymentManager, 200*time.Millisecond)
```

Comment:
The `namespace` is used to query model deployments as in the manager
`200ms` may be a bit optimistic